src/main/java/org/parabot/Landing.java

```diff
@@ -30,7 +30,7 @@ public static void main(String... args) {
 //        Thread.setDefaultUncaughtExceptionHandler(new FileExceptionHandler(ExceptionHandler.ExceptionType.CLIENT));
 
         if (Context.getJavaVersion() >= 9) {
-            UILog.log("Parabot", "Parabot doesn't support Java 9+ currently. Please downgrade to Java 8 to ensure Parabot is working correctly.");
+            UILog.log("Parabot", "Parabot doesn't support Java 9+ currently. Get Java 8 at adoptopenjdk.net to ensure Parabot is working correctly.");
```

Comment:
I would rather not provide any source or resource other than Java.com itself.
Neither do the users need JDK, as JRE is good enough.

Reply:
The reason I actually wanted to make this change is I had a number of users not knowing where to install Java 8 from at Java.com. It's now a much more buried feature, whereas OpenJDK is a super easy installation.

Reply:
The thing is, with Java.com I know I'm in good hands and the chance of that site being compromised is a lot lower than a third party site.
Besides, adoptopenjdk.net provides a different download than Java JRE itself.

Am I missing how easy it is to install? Go to Java.com and click download should be efficient enough. Unless the user already has a JDK installed, then they should download a different JDK, that is a little more tricky I agree.

Reply:
I think I'm getting a different screen because I'm on Linux. Can you confirm Java 8 is within a few clicks of being on Java.com? I'll close this PR if that's the case.

Reply:
https://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html

Googling jre 8

Reply:
https://files.ketelaar.me/ldzeh-t1kij-uke0s.mp4

Still feel free to adjust to PR to redirect to Java.com instead of adoptopenjdk.net